src/terms/pairwise.jl

```diff
@@ -0,0 +1,138 @@
+@doc raw"""
+Pairwise terms: Pairwise potential between nuclei, e.g., Van der Waals potentials, such as
+Lennard—Jones terms.
+The potential is dependent on the distance between to atomic positions and the pairwise
+atomic types:
+For a distance `d` between to atoms `A` and `B`, the potential is `V(d, params[(A, B)])`.
+The energy is of the form
```

Comment:
This formula is incorrect, and it doesn't reflect the fact that we consider periodic interactions. Might as well remove it